.github/workflows/macos-cmake-latest.yml

```diff
@@ -0,0 +1,133 @@
+name: Build pwsafe with CMake on macOS
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    env:
```

Comment:
Why need this section if we're setting most of the same parameters again on lines 36-45?